maintainers/maintainer-list.nix

```diff
@@ -619,6 +619,12 @@
     githubId = 4296804;
     name = "Alex Franchuk";
   };
+  afresquet = {
+    email = "29437693+afresquet@users.noreply.github.com";
```

Comment:
I'm not too sure about this, but I don't think GitHub's noreply emails are allowed. It seems like [Ofborg](https://gist.github.com/GrahamcOfBorg/0fbdb21923752d8e82076b6a58b94206#file-ofborg-eval-lib-tests-L86) isn't happy about this either, so I think it would be best to provide a normal email or just omit the email if you don't want to share your email.